cmd/eksctl/create.go

```diff
@@ -67,7 +67,8 @@ func createClusterCmd() *cobra.Command {
 	fs.StringVarP(&cfg.ClusterName, "name", "n", "", fmt.Sprintf("EKS cluster name (generated if unspecified, e.g. %q)", exampleClusterName))
 
 	fs.StringVarP(&cfg.Region, "region", "r", DEFAULT_EKS_REGION, "AWS region")
-	fs.StringVarP(&cfg.Profile, "profile", "p", "", "AWS creditials profile to use (overrides the AWS_PROFILE environment variable)")
+	fs.StringVarP(&cfg.Profile, "profile", "p", "", "AWS credentials profile to use (overrides the AWS_PROFILE environment variable)")
+	fs.StringToStringVarP(&cfg.Tags, "tags", "", map[string]string{}, "Custom tags")
```

Comment:
Could we change the description slightly and include an example? Something like the kops description might be good:
`A list of KV pairs used to tag the AWS resources (eg "Owner=John Doe,Team=Some Team"). `
Do you think its worth also putting in the description (or the readme) thats its limited to the CloudFormation stacks at present?